crates/universal-sierra-compiler-api/src/lib.rs

```diff
@@ -37,49 +36,47 @@ pub struct AssembledCairoProgramWithSerde {
     pub hints: Vec<(usize, Vec<Hint>)>,
 }
 
-pub fn compile_sierra_to_casm(sierra_program: &Program) -> Result<AssembledProgramWithDebugInfo> {
-    let assembled_with_info_raw = compile_sierra(
-        &serde_json::to_value(sierra_program)?,
-        None,
-        &SierraType::Raw,
-    )?;
-    let assembled_with_info: AssembledProgramWithDebugInfo =
-        serde_json::from_str(&assembled_with_info_raw)?;
+pub trait UscOutputTransformer {
```

Comment:
Nit, maybe rename it to just `UscOutput`

Function could still be named `transform`, but probably something like `convert` could work as well.

Reply:
Renamed to `UniversalSierraCompilerOutput` and `convert`